docs/phdata.rst

```diff
@@ -483,26 +555,92 @@ Also, if some information is not available the relative field may be omitted.
+
+.. _exp_settings:
+
+Experimental Settings User Group
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+It is often desirable to store additional information and/or explanation of the experiment.
+Such information is best stored in a ``user/experimental_settings`` group. All information in this
+group should be stored in a way that is self-explanatory, with human-readable details included.
+
+If :ref:`marker IDs<marker_ids>`) are included in ``/photon_data/detectors``,
+``/user/experimental_settings/`` is the preferred location to specify the significance
+of each marker ID.
+
+If values such as FRET correction factors are stored, the specific definition and method by which
+they were determined should be included as well so that interpretation can be made unambiguous.
+We suggest using the definition in `Hellenkamp et. al. 2018 <https://doi.org/10.1038/s41592-018-0085-0>`_
+
+See <link to notebook to be created> for an example of how to use the */user/experimental_settings/**
+group.
+
 
 .. _glossary:
 
 Additional notes and definitions
 --------------------------------
 
-.. _detector_ids:
+.. _record_ids:
 
-Detector pixel IDs
-^^^^^^^^^^^^^^^^^^
+Record IDs
+^^^^^^^^^^
 
-A *detector pixel ID* (or simply *pixel ID*) is the "name" of each pixels and
-is usually a single integer. Pixels are normally numbered incrementally, but not necessarily so.
+A *record ID* is the "name" for each type of event recorded in ``/photon_data/detectors``.
+These are divided into two broad types: "detector IDs" and "marker IDs".
+In previous versions these were sometimes also referred to as pixel IDs
+Simply put a detector ID corresponds to an event that results from the arrival of a photon
+at one of the detectors or pixels.
+A "marker ID" is any other type of event (e.g. sync signal, FLIM marker) recorded
+in the original source file/by the TCSPC card.
+
+Record IDs are usually a single integer.
+Detectors are normally numbered incrementally, but are not required to be.
 In other words, a file containing data taken with 2 single-point (pixel) detectors could have
 the first detector labeled "4" and the second detector labeled "6".
-In some cases (when using detector arrays) the pixel ID
+In some cases (when using detector arrays) the record ID
 can be a *n*-tuple of integers. This allow to specify, for each pixel,
 the module number and the X, Y location, for example. Therefore, an
-array of pixel IDs can be either a 1-D column array or a 2-D array.
-In either cases, each row identifies a pixel.
+array of record IDs can be either a 1-D column array or a 2-D array.
+In either case, each row identifies a record.
+
+.. _marker_ids:
+
+Marker IDs
+""""""""""
+
+If the acquisition software also records events such as sync signals or markers indicating
+a change in position of a piezo or galvo scanner, these events should also be assigned a unique
+pixel ID. These events are all considered markers. To distinguish between detector (i.e. real photons)
```

Comment:
Might I suggest "a unique record ID" (or whatever we decide on if my other comment gets any traction) and not "pixel ID" here, because the use of "pixel" to mean a detector element in this raster-scan-related context is potentially confusing?